DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Bindings/RhinoSendBinding.cs

```diff
@@ -137,22 +137,28 @@ public async Task Send(string modelCardId)
     using var unitOfWork = _unitOfWorkFactory.Resolve<SendOperation<RhinoObject>>();
     try
     {
-      // 0 - Init cancellation token source -> Manager also cancel it if exist before
-      CancellationTokenSource cts = _cancellationManager.InitCancellationTokenSource(modelCardId);
-
-      // 1 - Get model
       if (_store.GetModelById(modelCardId) is not SenderModelCard modelCard)
       {
+        // Handle as GLOBAL ERROR at BrowserBridge
```

Comment:
I believe it was necessary to define behavior if model card is not found. Maybe place is wrong but it is something developers keep in mind to handle it how, because it will effect the showing error on UI.